core/test/qa_DynamicBlock.cpp

```diff
@@ -14,7 +14,7 @@ struct fixed_source : public gr::Block<fixed_source<T>, gr::PortOutNamed<T, "out
 
     gr::work::Result
     work(std::size_t requested_work) {
-        auto &port   = gr::outputPort<0>(this);
+        auto &port   = gr::outputPort<0, gr::traits::port::port_kind::Stream>(this);
```

Comment:
What about `gr::traits::port::port_kind::Stream` -> `gr::port::kind::Stream` or `gr::port_kind::Stream` for easier readability?